index.js

```diff
@@ -234,10 +240,31 @@ function updateDefinitionsSpec(mongooseModels) {
   }
 }
 
+/**
+ * @description Generates tags spec
+ *
+ * @returns Tags spec
```

Comment:
The function does not return anything as far as I'm aware:D

Reply:
Sorry 🤣 Just removed that @returns from updateTagsSpec and updateDefinitionsSpec.

Reply:
No need to apologize:D leggo 🚀